src/index.ts

```diff
@@ -278,6 +278,9 @@ export function register (options: Options = {}): Register {
       },
       getDirectories: getDirectories,
       directoryExists: directoryExists,
+      fileExists: ts.sys.fileExists,
+      readFile: ts.sys.readFile,
+      readDirectory: ts.sys.readDirectory,
```

Comment:
We have existing implementations of two of these functions, should we just use those?